docs/docs-content/clusters/edge/third-party-integrations/tailscale.md

```diff
@@ -0,0 +1,249 @@
+
+  ```yaml
+  cat << EOF > user-data
+  stylus:
+    site:
+      paletteEndpoint: api.spectrocloud.com
+      edgeHostToken: $token
+      projectName: Default
+      name: edge-randomid
+  install:
+    poweroff: true
+    bind_mounts:
+    - /var/lib/tailscale
+  users:
+    - name: kairos
+      passwd: kairos
+  EOF
+  ```
+
+6. Next, add a `stages` block to the **user-data** file to automatically enable Tailscale and register the Edge device. Replace `$AUTH-KEY` with your authorization key from Tailscale:
+
+  ```yaml {14}
+  stages:
+    boot.after:
+      - name: "Register device with Tailscale"
+        if: '[ ! -f "/run/cos/recovery_mode" ] && ! grep _current-profile /var/lib/tailscale/tailscaled.state'
+        commands:
+          - |
+            ID=$(cat /sys/class/dmi/id/product_uuid)
+            if [ -f /oem/tailscale/tailscaled.state ]; then
+              systemctl stop tailscaled
+              cp /oem/tailscale/tailscaled.state /var/lib/tailscale/tailscaled.state
+              systemctl start tailscaled
+              tailscale up --ssh --hostname="edge-${ID}"
+            else
+              tailscale up --authkey=$AUTH-KEY --ssh --hostname="edge-${ID}"
+              mkdir /oem/tailscale
+              cp /var/lib/tailscale/tailscaled.state /oem/tailscale/tailscaled.state
+            fi
+      - name: "Enable Tailscale"
+        if: '[ ! -f "/run/cos/recovery_mode" ] && grep _current-profile /var/lib/tailscale/tailscaled.state'
+        commands:
+          - |
+            ID=$(cat /sys/class/dmi/id/product_uuid)
+            tailscale up --ssh --hostname="edge-${ID}"
+  ```
+
+  If you already have a `stages` block in your user-data file, you must merge the existing block together with the above content. The `stages` block is based on Kairos cloud-init stages. For more information on cloud init stages, refer to [Cloud Init Stages](../edge-configuration/cloud-init.md).
+
+  :::info
+
+  In the above `stages` block, you are using the device ID of your Edge device (read from `/sys/class/dmi/id/product_uuid`) as the hostname with which to register your device with Tailscale. For more information about how this ID is generated, refer to [Install Configurations](../edge-configuration/installer-reference.md#device-id-uid-parameters).
+
+  If you want to use a different hostname, especially when using the `deviceUIDPaths` parameter in the user-data, you can adjust the two `ID=$(cat /sys/class/dmi/id/product_uuid)` lines in the content above to match your custom device naming configuration.
+
+  :::
+
+7. Build the Edge device installation ISO and providers images. Afterward, push the provider images to an image registry. For more information, refer to [Build Edge Artifacts](../edgeforge-workflow/palette-canvos.md).
+
+8. Flash your external volume with the Edge installer ISO image. We recommend using [balena etcher](https://etcher.balena.io/) to flash your volume. 
```

Comment:
Is there a specific reason for why we would recommend this specific tool versus just mentioning it as an example? In other Edge docs, we mention it as an option, but we stay away from tooling recommendations unless there is a good reason for it.